### PR TITLE
Add system network stats (interface stats) to Dr.Kafka

### PR DESCRIPTION
**Pull request by ambud**
Add system network stats (interface stats) to Dr.Kafka:
- can be used for more accurate load calculation during rebalancing
- apply network load limits to Kafka brokers for partition assignment

Cleaning up unused imports / code
Fixing generics

### CODE REVIEW

File: kafkastats/src/main/java/com/pinterest/doctorkafka/stats/KafkaStatsMain.java [hunk 1]
```diff
@@ -122,14 +118,16 @@ public static void main(String[] args) throws Exception {
     String kafkaConfigPath = commandLine.getOptionValue(KAFKA_CONFIG);
     long uptimeInSeconds = Long.parseLong(commandLine.getOptionValue(UPTIME_IN_SECONDS));
     long pollingInterval = Long.parseLong(commandLine.getOptionValue(POLLING_INTERVAL));
+    String primaryNetworkInterfaceName = commandLine.getOptionValue(PRIMARY_INTERFACE_NAME, "eth0");
```
Comment (yuyang08) [inline kafkastats/src/main/java/com/pinterest/doctorkafka/stats/KafkaStatsMain.java]: declare "eth0" as a constant like `DEFAULT_PRIMARY_INTERFACE_NAME` at the beginning of the class?

Comment (ambud) [reply kafkastats/src/main/java/com/pinterest/doctorkafka/stats/KafkaStatsMain.java]: done

**Comment by yuyang08** [comment]
thanks for making the  improvement!

**Comment by ambud** [comment]
thanks @yuyang08

**Pull request merged**